vector/src/act/res/values/config.xml

```diff
@@ -111,4 +111,9 @@
 
     <bool name="use_progressbar_original_background">true</bool>
     <bool name="use_fab_color_as_accent_color">true</bool>
+    
+    <bool name="hide_message_view_source">true</bool>
```

Comment:
I'd suggest that we have positive configuration values e.g. `show_message_view_sources` as more often than not in the code you will then have:
```
    if (show_message_view_sources) {
      do show logic
    }
```
rather than:
```
    if (! hide_message_view_sources) {
      show logic
    }
```